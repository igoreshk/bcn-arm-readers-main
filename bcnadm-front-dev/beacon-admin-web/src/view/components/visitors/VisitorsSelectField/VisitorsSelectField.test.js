import React from 'react';
import { shallow } from 'enzyme';
import { VisitorsSelectField } from './index';

const props = {
  selectVisitorsIds: jest.fn(),
  visitors: [],
  translate: jest.fn(),
  isLoading: false,
  watcher: {
    entityId: 'someId',
    name: 'someName',
    visitorIds: []
  }
};

const wrapper = shallow(<VisitorsSelectField {...props} />);

describe('VisitorsSelectField', () => {
  it('Check matching snapshot', () => expect(wrapper).toMatchSnapshot());
});

