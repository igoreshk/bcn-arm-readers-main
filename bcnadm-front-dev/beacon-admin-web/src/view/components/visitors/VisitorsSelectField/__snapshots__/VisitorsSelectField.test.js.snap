// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisitorsSelectField Check matching snapshot 1`] = `
<WithStyles(ForwardRef(FormControl))
  className="visitorSelectFormControl"
>
  <WithStyles(ForwardRef(FormLabel))
    disabled={true}
  >
    <span
      className="placeholder"
    />
  </WithStyles(ForwardRef(FormLabel))>
</WithStyles(ForwardRef(FormControl))>
`;
